### PR TITLE
Revert globalDescriptorSet has not binding for v3.4.0

### PR DESCRIPTION
**Pull request by troublemaker52025**

### CODE REVIEW

**Comment by troublemaker52025** [comment]
* https://github.com/cocos-creator/3d-tasks/issues/10138

File: cocos/renderer/pipeline/shadow/ShadowStage.cpp [hunk 1]
```diff
@@ -64,7 +64,6 @@ void ShadowStage::render(scene::Camera *camera) {
         return;
     }
 
-    _pipeline->getPipelineUBO()->updateShadowUBO(camera);
```
Comment (troublemaker52025) [inline cocos/renderer/pipeline/shadow/ShadowStage.cpp:67]: 重复更新了

File: cocos/renderer/pipeline/shadow/ShadowStage.cpp [hunk 2]
```diff
@@ -64,9 +64,8 @@ void ShadowStage::render(scene::Camera *camera) {
...
     auto *cmdBuffer = _pipeline->getCommandBuffers()[0];
-
+    _pipeline->getPipelineUBO()->updateShadowUBOLight(_globalDS, _light);
```
Comment (troublemaker52025) [inline cocos/renderer/pipeline/shadow/ShadowStage.cpp:68]: 使用不绑定贴图的 shadowUBO更新函数

**Review by jk20012001 — APPROVED** [review]

**Comment by troublemaker52025** [comment]
* https://github.com/cocos-creator/engine/pull/9730

**Review by YunHsiao — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**